shoop/xtheme/templates/shoop/xtheme/_model_widget.jinja

```diff
@@ -0,0 +1,20 @@
+<select id="id_{{ name }}" name="{{ name }}" class="xtheme-model-choice-widget">
+    {% for pk, object in objects %}
+        {% if pk %}
+            {% set url = shoop_admin.model_url(object, "detail") %}
```

Comment:
Jinja supports ternary expressions; `{% set url = (shoop_admin.model_url(...) if pk else "") %}` is arguably clearer.